Expensieve/Expensieve
# Lucascontreras/fe service dashboard

## Overview

**Connected the backend and frontend**


**Description**
first request is implemented, currently getting "all data" on App.tsx

**Steps to Validate Feature**

1. Run `start-dev`
2. Data from the backend should be added to state on the App component (confirm with React Dev Tools)
